.github/workflows/release.yml

```diff
@@ -84,11 +84,27 @@ jobs:
         uses: google-github-actions/setup-gcloud@v2
         with:
           version: "469.0.0"
-      - name: Upload zip
+      - name: Upload zip to GCP
         uses: google-github-actions/upload-cloud-storage@v2
         with:
           path: ${{ env.OUTNAME }}.zip
           destination: ${{ secrets.GCP_BUCKET }}/${{ github.ref_name }}/
+      - name: Copy build
+        shell: bash
+        run: |
+          mkdir build-zip
+          cp ${{ env.OUTNAME }}.zip build-zip
+      - name: Upload zip to R2
+        uses: jakejarvis/s3-sync-action@master
```

Comment:
Using `@master` seems dangerous, any commit to the repository might break our workflow. Also that action seems to have been abandoned by its author ~ 5 years ago, are there alternatives?